packaging/installer/methods/kickstart-64.md

```diff
@@ -71,6 +71,26 @@ your installation. Here are a few important parameters:
     kickstart run the process using those files. This option conflicts with the `--stable-channel` option. If you set
     this _and_ `--stable-channel`, Netdata will use the local files.
 
+### Claim node to Netdata Cloud during installation
+
+The `kickstart.sh` script accepts additional parameters to automatically [claim](/claim/README.md) your node to Netdata
+Cloud immediately after installation. Find the `token` and `rooms` strings by [signing in to Netdata
+Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces), then clicking on **Claim Nodes** in the [Spaces management
+area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces).
+
+- `--claim-token`: The unique token associated with your Space in Netdata Cloud.
+- `--claim-rooms`: A comma-separated list of tokens for each War Room this node should appear in.
+- `--claim-proxy`: Should take the form of `socks5[h]://[user:pass@]host:ip` for a SOCKS5 proxy, or
+  `http://[user:pass@]host:ip` for an HTTP(S) proxy.See [claiming through a
+  proxy](/claim/README.md#claim-through-a-proxy) for details.
+- `--claim-url`: Defaults to `https://app.netdata.cloud`.
+
+For example:
+
+```bash
+bash <(curl -Ss https://my-netdata.io/kickstart-static64.sh) --claim-token=TOKEN --claim-rooms=ROOM1,ROOM2
```

Comment:
Let's add 2 screenshots here:
I will send them in comments on this GH issue

Reply:
And a paragraph that says:
1) Visit app.netdata.cloud
2) Login/signup
3) Click on Claim Node
4) Find the tokens in that field

In general a screenshot with where the claim node is, is super useful. I remember users having a hard time to discover it.

Reply:
Hey, I opted to merge without this change. Here's why. I don't want to add Cloud screenshots to a Netdata installation method, as I think it confuses new users about the separation between the two. Also, the text here already explains to sign in and what to click. In addition, the [Spaces management area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces) link has essentially the same screenshot to show users where to find it.

Reply:
Hey ,I should be less lazy and checked the spaced management area link before commenting. 

Sorry for the noise

Reply:
No worries, I just wanted to be clear that I wasn't ignoring you!